modules/pam_listfile/pam_listfile.c

```diff
@@ -307,6 +316,7 @@ pam_listfile(pam_handle_t *pamh, int argc, const char **argv)
     assert(PAM_SUCCESS == 0);
     assert(PAM_AUTH_ERR != 0);
 #endif
+    inf = fdopen(fd,"r");
```

Comment:
`fdopen(3)` can theoretically fail

Reply:
Indeed, thanks for catching that. I have now changed the code to take into account that fdopen can fail.